hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/GenesisRecordsConsensusHook.java

```diff
@@ -160,6 +178,19 @@ private void createAccountRecordBuilders(
                     Transaction.newBuilder().body(TransactionBody.newBuilder().cryptoCreateAccount(txnBody));
             recordBuilder.transaction(txnBuilder.build());
 
+            var balance = account.tinybarBalance();
+            if (balance != 0) {
+                var accountID = AccountID.newBuilder()
+                        .accountNum(account.accountId().accountNumOrElse(0L))
+                        .shardNum(account.accountId().shardNum())
+                        .realmNum(account.accountId().realmNum())
+                        .build();
+
+                recordBuilder.transferList(TransferList.newBuilder()
+                        .accountAmounts(asAccountAmounts(Map.of(accountID, balance)))
+                        .build());
```

Comment:
We should not be building transferList here

Reply:
@Neeharika-Sompalli is initial funding of the treasury not considered a transfer?

Reply:
@mhess-swl  we do like this in mono-service. In mod-service I was thinking we don't hard code transferList anywhere. May be we should invoke FinalizeRecordHandler? @tinker-michaelj  thoughts ?

Reply:
May be this is the only way we can do now.